subsys/bluetooth/host/conn_internal.h

```diff
@@ -107,6 +107,7 @@ struct bt_conn {
 	bt_security_t		sec_level;
 	bt_security_t		required_sec_level;
 	u8_t			encrypt;
+	u8_t                    force_pairing;
```

Comment:
Why not add it to `conn->flags` instead? That way you don't need to increase the struct size.

Reply:
Good idea. Switched to that instead.